Conceptpower+Spring/src/test/java/edu/asu/conceptpower/app/core/impl/RequestsManagerTest.java

```diff
@@ -19,22 +27,43 @@
  
     @InjectMocks
     private RequestsManager requestManager;
- 
+    
+    private ReviewRequest request = new ReviewRequest();
+    
     @Before
     public void setup() {
         MockitoAnnotations.initMocks(this);
         
+        List<ReviewRequest> responseList = new ArrayList<>();
+        ReviewRequest updatedRequest = new ReviewRequest();
+        request.setId("REVIEW1234-thisissupposedtobesecret");
+        
+        updatedRequest.setId("REVIEW1234-thisissupposedtobesecret2");
+        updatedRequest.setConceptId("WID-10126926-N-05-Einstein");
+        updatedRequest.setRequest("Review the Spelling again");
+        updatedRequest.setRequester("admin");
+        updatedRequest.setResolver("admin");
+        updatedRequest.setCreatedAt(OffsetDateTime.now());
+        updatedRequest.setStatus(ReviewStatus.OPENED);
+        
+        responseList.add(updatedRequest);
+        Mockito.when(dbClient.getReview("REVIEW1234-thisissupposedtobesecret2")).thenReturn(updatedRequest);
+        
+        Mockito.when(dbClient.getAllReviews("WID-10126926-N-05-Einstein")).thenReturn(responseList);
     }
     
     @Test
     public void test_addReviewRequest_success() {
         
         ReviewRequest reviewRequest = new ReviewRequest();
         reviewRequest.setStatus(ReviewStatus.OPENED);
+        reviewRequest.setId("REVIEW1234-thisissupposedtobesecret");
         reviewRequest.setRequester("admin");
         reviewRequest.setConceptId("WID-01006675-N-02-mental_test");
         reviewRequest.setRequest("Testing Method");
            
+        Mockito.when(dbClient.getReview("REVIEW1234-thisissupposedtobesecret")).thenReturn(null);
```

Comment:
why is this here? the default value is null in mockito if you haven't defined it otherwise.

Reply:
?